services/user-service/middleware/authorization.js

```diff
@@ -11,27 +11,24 @@ const verifySessionToken = (req, res, next) => {
 
```

Comment:
Need to implement the new error handler for this middleware file. Next PR will have it.